### PR TITLE
 add behat environment

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: behat.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ default:
         Behat\Symfony2Extension:
         Behat\MinkExtension:
             default_session: selenium2
-            base_url: 'http://openorchestra.test:8000/app_test.php'
-            browser_name: chrome
+            base_url: 'http://admin.openorchestra.dev/app_behat.php'
```
Comment (nicolasThal) [inline behat.yml]: you should let the previous url, this way we can execute the behat test on the integration server

File: behat.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ default:
...
+            browser_name: firefox
```
Comment (nicolasThal) [inline behat.yml]: it is better to use chrome I think.

ask @itkg-bfouche to confirm !

File: web/app_behat.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Debug\Debug;
+
+// If you don't want to setup permissions the proper way, just uncomment the following PHP line
```
Comment (nicolasThal) [inline web/app_behat.php]: do not commit commented code

File: app/AppKernel.php [hunk 4]
```diff
@@ -50,7 +50,7 @@ public function registerBundles()
             new Solution\MongoAggregationBundle\SolutionMongoAggregationBundle(),
         );
 
-        if (in_array($this->getEnvironment(), array('dev', 'test'))) {
+        if (in_array($this->getEnvironment(), array('dev', 'test', 'behat'))) {
```
Comment (itkg-ngilain) [inline app/AppKernel.php]: Are debug tools required on the functionnal testing environnement or is it better to use those tests in a production-like environnement ?

File: app/config/config_behat.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+imports:
+    - { resource: config_dev.yml }
```
Comment (itkg-ngilain) [inline app/config/config_behat.yml]: same question here

File: app/AppKernel.php (hunk 4 above)
Comment (nicolasThal) [inline app/AppKernel.php]: As the web_profiler toolbar is disabled, i would suggest to only use the production bundles

**Comment by nicolasThal** [comment]
:+1:

**Comment by itkg-nanne** [comment]
:+1:

File: web/app_behat.php [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Symfony\Component\HttpFoundation\Request;
+
+$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
```
Comment (itkg-ngilain) [inline web/app_behat.php]: $loader is never used, is it needed ?

Comment (nicolasThal) [inline web/app_behat.php]: at least the require_once is.

For the loader, it can be used for the ApcLoader

**Pull request merged**